test/apps/scroll/test.ts

```diff
@@ -1,107 +1,133 @@
+import {build} from '../../../api';
+import {AppRunner} from '../AppRunner';
 
-describe('scroll', function() {
-	this.timeout(10000);
-
-	let r: AppRunner;
-
-	// hooks
-	before('build app', () => build({ cwd: __dirname }));
-	before('start runner', async () => {
-		r = await new AppRunner().start(__dirname);
-	});
-
-	after(() => r && r.end());
-
-	// tests
-	it('scrolls to active deeplink', async () => {
-		await r.load('/tall-page#foo');
-		await r.sapper.start();
-
-		const scrollY = await r.page.evaluate(() => window.scrollY);
-		assert.ok(scrollY > 0, String(scrollY));
-	});
-
-	it('scrolls to any deeplink if it was already active', async () => {
-		await r.load('/tall-page#foo');
-		await r.sapper.start();
-
-		let scrollY = await r.page.evaluate(() => window.scrollY);
-		assert.ok(scrollY > 0, String(scrollY));
-
-		scrollY = await r.page.evaluate(() => {
-			window.scrollTo(0, 0)
-			return window.scrollY
-		});
-		assert.ok(scrollY === 0, String(scrollY));
-
-		await r.page.click('[href="tall-page#foo"]');
-		scrollY = await r.page.evaluate(() => window.scrollY);
-		assert.ok(scrollY > 0, String(scrollY));
-	});
-
-	it('resets scroll when a link is clicked', async () => {
-		await r.load('/tall-page#foo');
-		await r.sapper.start();
-		await r.sapper.prefetchRoutes();
-
-		await r.page.click('[href="another-tall-page"]');
-		await r.wait();
-
-		assert.equal(
-			await r.page.evaluate(() => window.scrollY),
-			0
-		);
-	});
-
-	it('preserves scroll when a link with sapper-noscroll is clicked', async () => {
-		await r.load('/tall-page#foo');
-		await r.sapper.start();
-		await r.sapper.prefetchRoutes();
-
-		await r.page.click('[href="another-tall-page"][sapper-noscroll]');
-		await r.wait();
-
-		const scrollY = await r.page.evaluate(() => window.scrollY);
-
-		assert.ok(scrollY > 0);
-	});
-
-	it('scrolls into a deeplink on a new page', async () => {
-		await r.load('/tall-page#foo');
-		await r.sapper.start();
-		await r.sapper.prefetchRoutes();
-
-		await r.page.click('[href="another-tall-page#bar"]');
-		await r.wait();
-		assert.equal(await r.text('h1'), 'Another tall page');
-		const scrollY = await r.page.evaluate(() => window.scrollY);
-		assert.ok(scrollY > 0);
-	});
-
-	it('scrolls to a deeplink on a new page no matter the previous scroll position', async () => {
-		await r.load('/a-third-tall-page#top');
-		await r.sapper.start();
-		await r.sapper.prefetchRoutes();
-
-		await r.page.click('a#top');
-		await r.wait();
-		const firstScrollY = await r.page.evaluate(() => window.scrollY);
-
-		await r.load('/a-third-tall-page#bottom');
-		await r.sapper.start();
-		await r.sapper.prefetchRoutes();
-
-		await r.page.click('a#bottom');
-		await r.wait();
-		const secondScrollY = await r.page.evaluate(() => window.scrollY);
-
-		assert.equal(firstScrollY, secondScrollY);
-	});
-
-	it('survives the tests with no server errors', () => {
-		assert.deepEqual(r.errors, []);
-	});
+describe('scroll', function () {
```

Comment:
Please don't reformat all the code as part of a PR. Our linting rules aren't really enforced right now, but the existing style is the style we want to keep.

Reply:
Yeah, I am trying to fix this now. I guess my editor used global default Prettier settings since this project has no config.

Reply:
Alright, I reverted the spacing changes. Sorry about that.